### PR TITLE
feat(python): IAST weak hash

### PR DESCRIPTION
**Pull request by avara1986**
## Description

Update tests and docker projects to enable IAST weak hash for Python

## Check list

Your PR is not ready to be reviewed? Please save it as a draft :pray:

- [x] Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
- [x] CI is passing
- [x] There is at least one approval from code owners

Yes to all? Feel free to merge it whenever you want :heart:

### CODE REVIEW

**Comment by robertomonteromiguel** [comment]
python django and uwsgi build failed:

`  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "/usr/local/lib/python3.9/site-packages/ddtrace/internal/module.py", line 190, in _exec_module
    self.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/app/urls.py", line 6, in <module>
    from iast import weak_hash, secure_algorithm
ModuleNotFoundError: No module named 'iast'
weblog exited with code 137
`

**Review by juanjux — APPROVED** [review]

**Review by gnufede — APPROVED** [review]

**Comment by cbeauchesne** [comment]
Hi @avara1986, is this PR not a draft anymore, can I review it ?

**Comment by avara1986** [comment]
> Hi @avara1986, is this PR not a draft anymore, can I review it ?

Yes @cbeauchesne , I updated to "ready to review" :smile:

File: tests/appsec/iast/test_iast.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
     java={"spring-boot": "0.108.0", "spring-boot-jetty": "0.108.0", "*": "?"},
     nodejs={"express4": "3.6.0", "*": "?"},
     php_appsec="?",
-    python="?",
+    python="1.6.0rc1.dev",
```
Comment (cbeauchesne) [inline tests/appsec/iast/test_iast.py:15]: This version does not exists yet. It means that the test is not executed. And once it'll exists, if the tracers does not pass the test, your CI will be broken (among a lot of other CI).

FYI, current dev version from your `main` branch is python@1.5.0rc2.dev105+g61f01775 : https://github.com/DataDog/system-tests/actions/runs/3386289779/jobs/5625523608#step:9:31

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

**Review by juanjux — APPROVED** [review]

**Comment by cbeauchesne** [comment]
For a reason I ignore, GH did not run action on the last commit. I manually ran them : https://github.com/DataDog/system-tests/actions/runs/3472503367

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**